libs/generators/src/lib/generators/generators.component.ts

```diff
@@ -1,11 +1,16 @@
 import { CommonModule } from '@angular/common';
 import { Component } from '@angular/core';
+import { RouterLink, RouterLinkActive, RouterOutlet } from '@angular/router';
 
 @Component({
   selector: 'cli-generators',
   standalone: true,
-  imports: [CommonModule],
+  imports: [CommonModule, RouterLink, RouterLinkActive, RouterOutlet],
   templateUrl: './generators.component.html',
-  styleUrls: ['./generators.component.css'],
+  styleUrls: ['./generators.component.scss'],
```

Comment:
should be onPush, @ronnetzer is it not the default?

Reply:
@Danieliverant right, didn't notice.